components/button/button.html

```diff
@@ -0,0 +1,278 @@
+                        <iframe src="https://carbon.now.sh/embed/cHUOSDS10vYrpZglMtnN"
+                            style="width: 100%; height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                    <div class="separator"></div>
+
+                    <!--Float button-->
+                    <div class="content-subhead">
+                        Float Button
+                    </div>
+                    <div class="content-subtitle">
+                        A demo of the use of <code>float button</code> is shown in the bottom-right corner
+                        of the page. Just add the <code>btn-float</code> class to the button.
+                    </div>
+                    <div class="component-container">
+                        <button class="btn-float">
+                            <svg aria-hidden="true" focusable="false" height="2em" data-prefix="fas"
+                                data-icon="arrow-up" class="svg-inline--fa fa-arrow-up fa-w-14" role="img"
+                                xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+                                <path fill="currentColor"
+                                    d="M34.9 289.5l-22.2-22.2c-9.4-9.4-9.4-24.6 0-33.9L207 39c9.4-9.4 24.6-9.4 33.9 0l194.3 194.3c9.4 9.4 9.4 24.6 0 33.9L413 289.4c-9.5 9.5-25 9.3-34.3-.4L264 168.6V456c0 13.3-10.7 24-24 24h-32c-13.3 0-24-10.7-24-24V168.6L69.2 289.1c-9.3 9.8-24.8 10-34.3.4z">
+                                </path>
+                            </svg>
+                            <a href="#"></a>
+                        </button>
+                    </div>
+                    <div class="iframe">
+                        <iframe src="https://carbon.now.sh/embed/V0DCkOo0RQPl1KE6fC6P"
+                            style="width: 100%; height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+            </section>
+        </main>
+    </div>
+
+    <!--Float button demo-->
+    <button class="btn-float-demo">
+        <svg aria-hidden="true" focusable="false" height="2em" data-prefix="fas" data-icon="arrow-up"
+            class="svg-inline--fa fa-arrow-up fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 448 512">
+            <path fill="currentColor"
+                d="M34.9 289.5l-22.2-22.2c-9.4-9.4-9.4-24.6 0-33.9L207 39c9.4-9.4 24.6-9.4 33.9 0l194.3 194.3c9.4 9.4 9.4 24.6 0 33.9L413 289.4c-9.5 9.5-25 9.3-34.3-.4L264 168.6V456c0 13.3-10.7 24-24 24h-32c-13.3 0-24-10.7-24-24V168.6L69.2 289.1c-9.3 9.8-24.8 10-34.3.4z">
+            </path>
+        </svg>
+        <a href="#top"></a>
+    </button>
+    <script>
+        const body = document.querySelector('body'),
```

Comment:
can we keep script.js folder separate:

Reply:
Hey @saketkum11, yes I will do that, thank you for the suggestion. But since I would be making the changes to all the components, I thought it would be better to create a separate branch for these fix